src/components/header/header.js

```diff
@@ -0,0 +1,24 @@
+//Navbar.js
+import React from 'react';
+
+import '../header/header.css'
+import logo from '../images/applogo.png'; 
+import {FaInfoCircle } from 'react-icons/fa';
+
+import Popup from "reactjs-popup";
+
+import Content from "../popup/entryPopUp";
+
+const Header=() => {
+  return (
+    <header className='navbar'>
+      
+        <div className='navbar__title navbar__item'><img src={logo} /></div>
+        <div className='navbar__item'> </div>
+        <div className='navbar__item'> <Popup modal trigger={<button className="info_button"><FaInfoCircle /> Office Re-Entery Requirements</button>}>
+        {close => <Content close={close} />}
+      </Popup></div>
+    </header>
```

Comment:
consider cleanup divs?
```suggestion
      
        <div className='navbar__title navbar__item'><img src={logo} /></div>
        <div className='navbar__item'> </div>
        <div className='navbar__item'> 
        <Popup modal trigger={<button className="info_button"><FaInfoCircle /> Office Re-Entery Requirements</button>}>
        {close => <Content close={close} />}
      </Popup>
      </div>
    </header>
```